i18n/zh/docusaurus-plugin-content-docs/current/08-plugins/vault.md

```diff
@@ -0,0 +1,147 @@
+# vault  Plugin
+
+This plugin installs [vault](https://www.vaultproject.io/) in an existing Kubernetes cluster using the Helm chart for you test or develop vault.
+This plugin installs vault with replicas:3 by default value
+
+### Usage
+
+```yaml
+tools:
+- name: hashicorp-vault
+  # name of the plugin
+  instanceID: default
+  options:
+    # need to create the namespace or not, default: false
+    create_namespace: true
+    repo:
+      # name of the Helm repo
+      name: hashicorp
+      # url of the Helm repo
+      url: https://helm.releases.hashicorp.com
+    # Helm chart information
+    chart:
+      # name of the chart
+      chart_name: hashicorp/vault
+      # release name of the chart
+      release_name: vault
+      # k8s namespace where Vault will be installed
+      namespace: hashicorp
+      # whether to wait for the release to be deployed or not
+      wait: true
+      # the time to wait for any individual Kubernetes operation (like Jobs for hooks). This defaults to 5m0s
+      timeout: 5m
+      values_yaml: |
+        global:
+          enabled: true
+        server:
+          affinity: ""
+          ha:
+            enabled: true
+            replicas: 3
+            raft:
+              enabled: true
+              setNodeId: true
+          namespaceSelector:
+            matchLabels:
+              injection: enabled
+```
+
+Currently, except for `values_yaml`, all the parameters in the example above are mandatory.
+
+After vault pods deployed  by the plugin vault, you can follow the instructions below to initialize the vault:
+Before you follow the instructions bellow, you must install `jq` tool: a command tool to analyze json
+
+If you work on MacOS:
+```
+brew install jq
+```
+
+If you work on linux OS like CentOS:
+```
+sudo yum install jq
+```
+
+If you work on Linux OS like Ubuntu:
+```
+sudo apt-get install jq
+```
+
+
+in the  command below, the variable $NAMESPACE you should replace with "hashicorp-vault" if you not modify the namespace variable.
+Otherwise, use the namespace name you replaced
+
+1.
+```
+   # Initialize vault-0 with one key share and one key threshold.
+   kubectl exec vault-0 -n $NAMESPACE -- vault operator init -key-shares=1 -key-threshold=1 -format=json > cluster-keys.json
+```
+2 .
+  ```
+   # Display the unseal key found in cluster-keys.json
+   cat cluster-keys.json | jq -r ".unseal_keys_b64[]"
+   ```
+3.
+```
+   # Create a variable named VAULT_UNSEAL_KEY to capture the Vault unseal key.
+   VAULT_UNSEAL_KEY=$(cat cluster-keys.json | jq -r ".unseal_keys_b64[]")
+```
+4.
+```
+   # Unseal vault-0 running on the vault-0 pod.
+   kubectl exec vault-0  -n $NAMESPACE -- vault operator unseal $VAULT_UNSEAL_KEY
+```
+you will see the above command's output like this, make sure the item "Initialized" value is "true" and the item "Sealed" value is "false"
```

Comment:
The first letter should be uppercase. Missing period(".") at the end.

Reply:
> The first letter should be uppercase. Missing period(".") at the end.

Done. And Corrected the grammar error by Grammly.